cpp/ql/src/semmle/code/cpp/exprs/Access.qll

```diff
@@ -141,8 +171,20 @@ class FieldAccess extends VariableAccess {
 }
 
 /**
- * A field access of the form `obj->field`. The type of `obj` is a pointer,
- * so this is equivalent to `(*obj).field`.
+ * A field access whose qualifier is a pointer to a class, struct or union.
+ * These typically take the form `obj->field`, but also includes many accesses
```

Comment:
s/includes/include ?

Reply:
Hmm, I'm not sure.  I've rephrased entirely to avoid the issue.